Source/IDefaultValueProvider.cs

```diff
@@ -45,12 +45,12 @@ namespace Moq
 {
 	/// <include file='IDefaultValueProvider.xdoc' path='docs/doc[@for="IDefaultValueProvider"]/*'/>
 	[EditorBrowsable(EditorBrowsableState.Advanced)]
```

Comment:
Since EmptyDefaultValueProvider is already public, I think it would be fine to make this interface discoverable too.